features/unattached_commands.feature

```diff
@@ -424,3 +440,98 @@ Feature: Command behaviour when unattached
           | kinetic | python3.10     |                         |
           # Lunar has a BIG error message explaining why this is a clear user error...
           | lunar   | python3.11     | --break-system-packages |
+
+
+    @series.all
+    @uses.config.machine_type.lxd-container
+    Scenario Outline: Warn users not to redirect/pipe human readable output
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run shell command `pro version | cat` as non-root
+        Then I will see the following on stderr
+        """
+        """
+        When I run shell command `pro version > version_out` as non-root
+        Then I will see the following on stderr
+        """
+        """
+        When I run shell command `pro status | cat` as non-root
+        Then I will see the following on stderr
+        """
+        WARNING: this output is intended to be human readable, and subject to change.
+        In scripts, prefer using machine readable data from the `pro api` command,
+        or use `pro status --format json`.
+        """
```

Comment:
Is there any way we can write a test like this?
```
        When I run shell command `pro status as non-root
        Then I will not see the following on stderr
        """
        WARNING: this output is intended to be human readable, and subject to change.
        In scripts, prefer using machine readable data from the `pro api` command,
        or use `pro status --format json`.
        """
```

Reply:
Here is the catch: we cannot :/
This thing will always be on stderr on tests because our step involves redirecting the output :D
Even this test that you highlighted wouldn't need the `| cat`  to work :D :D 
If you have any suggestion on how to improve those tests, please post - appreciate it :D :D :D

Reply:
I was just wondering if there was a straightforward way to mimic a tty :upside_down_face:, it looks like it's possible but we'd need support in pycloudlib first so I created https://github.com/canonical/pycloudlib/issues/307

But this PR is good in the mean time :D